src/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/ArtworkDetails.tsx

```diff
@@ -209,20 +213,12 @@ export const ArtworkDetails: React.FC<ArtworkDetailsProps> = ({
   }
 
   const deriveBackLinkTo = () => {
-    const defaultBackLink = artworkId
-      ? "/collector-profile/my-collection"
-      : "/sell"
+    const defaultBackLink = "/sell"
     let backTo = defaultBackLink
-    if (isFirstStep && artworkId) {
```

Comment:
Don't we still need this logic even if we change the first step? Because the first step is still there 🤔

Reply:
This is related to the back button which we don't have anymore on the first step (Artwork Details)